pipeline/trusted-resources/pkg/trustedtask/trustedtaskrun_test.go

```diff
@@ -395,27 +382,21 @@ func getSignedTask(unsigned *v1beta1.Task, signer signature.Signer) (*v1beta1.Ta
 }
 
 // Generate key files to tmpdir, set configMap and return signer
-func getSignerFromFile(t *testing.T, ctx context.Context, k8sclient kubernetes.Interface) (signature.Signer, error) {
+func getSignerFromFile(t *testing.T, ctx *context.Context, k8sclient kubernetes.Interface) (signature.Signer, error) {
```

Comment:
Generally contexts are not passed around by pointer - you generally don't want a sub-process modifying the context of a parent.

Typically something like this would be done by initializing this at the beginning of a test, similar to how you are setting up the test logger. Can we pull this into a common "initialize context" func instead?

Reply:
Got it, thanks!